### PR TITLE
Add support for Redmi Note 5 and Note 5 Prime

### PR DESCRIPTION
**Pull request by barni2000**
- Update xiaomi-riva's msm-id to use predefined id macros.
- Add xiaomi-ugglite (msm8917) and ugg (msm8940).

### CODE REVIEW

File: lk2nd/device/dts/msm8952/msm8917-xiaomi-riva.dts [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #include <lk2nd.dtsi>
 
 / {
-	qcom,msm-id = <QCOM_ID_MSM8917 0>, <308 0>, <309 0>;
+	qcom,msm-id = <QCOM_ID_MSM8917 0>, <QCOM_ID_MSM8217 0>, <QCOM_ID_MSM8617 0>;
```
Comment (TravMurav) [inline lk2nd/device/dts/msm8952/msm8917-xiaomi-riva.dts:9]: Maybe I've missed this before, but do we really need 3g versions of the soc here?

Comment (barni2000) [reply lk2nd/device/dts/msm8952/msm8917-xiaomi-riva.dts:9]: They are defined in downstream i can try a boot without them.

Comment (TravMurav) [reply lk2nd/device/dts/msm8952/msm8917-xiaomi-riva.dts:9]: qcom always defines all "firmware" skus of the chip in the downstream soc dtsi, but I'd prefer to not have them unless needed

Comment (barni2000) [reply lk2nd/device/dts/msm8952/msm8917-xiaomi-riva.dts:9]: It seems it works fine without it i will check ugglite also.

File: lk2nd/device/dts/msm8952/msm8917-mtp.dts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8917 0>;
+	qcom,board-id = <QCOM_BOARD_ID_MTP 0>;
+};
+
+&lk2nd {
+	xiaomi-ugglite {
+		model = "Xiaomi Redmi Note 5A (ugglite)";
```
Comment (TravMurav) [inline lk2nd/device/dts/msm8952/msm8917-mtp.dts:15]: Could you also add those devices to the devices.md doc?

Comment (barni2000) [reply lk2nd/device/dts/msm8952/msm8917-mtp.dts:15]: Oh sorry i have forgotten it, ofc.

**Comment by TravMurav** [comment]
Thanks!

**Pull request merged**